bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/templates/FormBrowser.java

```diff
@@ -64,17 +101,26 @@ public Control getControl() {
 		return container;
 	}
 
-	public void setText(String text) {
-		this.text = text;
+	public void setText(String text)
+	{
+		this.text = convertMarkdownToHtml(text);
 		if (formText != null)
 			try
 			{
-				formText.setText(text);
+				formText.setText(this.text);
 			}
 			catch (Exception e)
 			{
 				Status status = new Status(IStatus.ERROR, IDFCorePlugin.PLUGIN_ID, "Error parsing the template description", e); //$NON-NLS-1$
 				IDFCorePlugin.getPlugin().getLog().log(status);
 			}
 	}
+
+	private String convertMarkdownToHtml(String markdownText)
+	{
+		Parser parser = Parser.builder().build();
+		Node document = parser.parse(markdownText);
+		HtmlRenderer renderer = HtmlRenderer.builder().build();
```

Comment:
Add exception block, if something goes wrong with the markdown parser it should still need to display the original text

Reply:
@kolipakakondal 
the try-catch block was added, exception is writing by Logger. The default text is returned when something goes wrong:
https://github.com/espressif/idf-eclipse-plugin/blob/db760d63e4174b294d5c8753cebe7ffced8ccdc1/bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/templates/FormBrowser.java#L119-L134